File: Python/BubbleSort.py
# -*- coding = utf-8 -*-
# Python实现冒泡排序

def bubbleSort(arr):
    for i in range(1,len(arr)):
        for j in range(0,len(arr)-i):
            if arr[j] > arr[j+1]:
                arr[j],arr[j+1] = arr[j+1],arr[j]
    return arr

File: Go/BubbleSort.go
//Go 实现冒泡排序
package main

func bubbleSort(arr []int) []int {
    length := len(arr)
    for i := 0; i < length; i++ {
        for j := 0; j < length - i - 1; j++ {
            if arr[j] > arr[j+1] {
                arr[j],arr[j+1] = arr[j+1],arr[j]
            }
        }
    }
    return arr
}